plugin/evm/vm.go

```diff
@@ -103,6 +99,17 @@ var (
 	errInvalidNonce               = errors.New("invalid nonce")
 )
 
+// mayBuildBlockStatus denotes whether the engine should be notified
+// that a block should be built, or whether more time has to pass
+// before doing so. See VM's [mayBuildBlock].
+type mayBuildBlockStatus uint8
+
+const (
+	wait mayBuildBlockStatus = iota
+	conditionalWait
```

Comment:
Could you change these to include buildBlock in them, so that the code is more clear about what they represent? Maybe, `waitBuildBlock`, `conditionalWaitBuildBlock`, and `buildBlock`